src/components/HeroSection.tsx

```diff
@@ -1,51 +1,82 @@
-                                    <h3 className="text-[1.375rem] font-semibold pt-2"> Dr Jan Kowalski</h3>
-                                    <p>Specjalistka w dziedzinie UX Design</p>
-                                </div>
-                            </div>
-                            <Image src={ChatBot} alt="ChatBot-image" className="max-w-xs" />
-                        </div>
-                    </div>
+  return (
+      <section id="hero" className="flex flex-col items-center justify-center py-24 px-8 z-10">
+        <div className="flex flex-col max-w-screen-lg text-center gap-8 lg:gap-12  lg:mt-8">
+          <div>
+            <h1 className="text-[clamp(2.7rem,5cqw,5rem)] font-bold leading-tight">
+              {" "}
+              Znajdź swojego{" "}
+              <span className="text-colorPrimary">promotora!</span>
+            </h1>
+            <p className="text-[clamp(1.2rem,2cqw,1.75rem)] max-w-screen-md mx-auto pt-8">
+              {" "}
+              Nasz system rekomendacyjny dopasuje go do tytułu lub opisu twojej
+              pracy dyplomowej.{" "}
+            </p>
+          </div>
+          <div className="flex flex-col md:flex-row justify-center gap-4 text-[1.125rem]">
+            <Button
+              asChild
+              variant={"secondary"}
+              className="text-[1.125rem] px-6 py-[1.84rem] rounded-xl"
+            >
+              <Link className="flex justify-center items-center gap-2" href="">
+                Wypróbuj <ArrowUpRight size={32} />
+              </Link>
+            </Button>
+            <Button
+              asChild
+              variant={"outline"}
+              className="text-[1.125rem] px-5 py-6 md:px-6 md:py-7 rounded-xl bg-transparent"
+            >
+              <Link href="">Dowiedz się więcej</Link>
+            </Button>
+          </div>
+          <div className="flex  items-center flex-col justify-center lg:flex-row-reverse lg:items-start lg:gap-5 max-w-screen-lg text-textSecondary text-lg  text-left leading-snug ">
+            <div className="bg-backgroundChatUser text-textSecondary max-w-xs max-h-[8.25rem] rounded-3xl px-5 py-4 mt-4">
+              <p>
+                {" "}
+                Wpływ modernistycznego stylu w projektowaniu stron
+                internetowychna percepcję i doświadczenia użytkownika.
+              </p>
+            </div>
+            <div className="flex lg:flex-row lg:items-start items-center flex-col-reverse justify-center">
+              <div className="relative lg:mt-6">
+                <div className="relative container bg-backgroundChatBot max-w-xs h-auto rounded-3xl  px-5 py-4 mt-4">
+                  <div className="absolute left-1/2 top-[-1.25rem] lg:left-[96%] lg:top-1/3 w-0 h-0 border-l-[1.25rem] border-r-[1.25rem] border-b-[1.25rem] border-transparent border-b-colorChatBot lg:transform lg:rotate-90"></div>
```

Comment:
nie jestem fanem hardocodowania wartości jak 1.25 rem 😭

Reply:
Tak jak napisałem w opisie PR'a, ten div wydaje mi się pozostałością z tego jak była próba robienia jakichś animacji więc póki co to po prostu wykomentowałem w https://github.com/Solvro/web-promochator/pull/17/commits/e3773143b6862a5094b27f4d884752d12e00e390